### PR TITLE
adding my name

### PR DESCRIPTION
**Pull request by amin135**
@aneagoie Just added my name and Github profile as a new row. Please approve and merge if all is well.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -164,6 +164,21 @@ <h1 class="heading">Available Developers</h1>
 							</div>
 						</div>
 
+						<div class="row">
+					    <div class="cell" data-title="Full Name">
```
Comment (aneagoie) [inline index.html]: Can you fix the indentation here to match the rest of the rows?

**Comment by amin135** [comment]
@aneagoie Fixed the indentation. Thanks

**Comment by aneagoie** [comment]
Have a look at the file changes.. something is off. I recommend you create a new pull request after you pull the latest master!

**Comment by amin135** [comment]
will create new pull request per aneagoie's advice

**Pull request closed without merging**